security-proxy/src/main/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -44,8 +44,11 @@
         <to type="forward" last="true">/$1</to>
     </rule>
     <rule>
+      <from>^/(cas|analytics|catalogapp|downloadform|extractorapp|geonetwork|geoserver|geofence|geowebcache|header|ldapadmin|mapfishapp)$</from>
```

Comment:
downloadform will never be accessed manually, I think it's useless here but that's also OK to have it.